packages/appsync-emulator-serverless/lambdaRunnerPython.js

```diff
@@ -0,0 +1,56 @@
+const {
+  log,
+  sendErr,
+  sendOutput,
+  installExceptionHandlers,
+} = require('./lambda/util');
+
+process.once(
+  'message',
+  async ({ serverlessDirectory, handlerMethod, payload }) => {
+    try {
+      log.info('load', handlerMethod);
+
+      const { spawn } = require('child_process');
+      const args = ['invoke', 'local', '-f', handlerMethod];
+
+      const sls = spawn('sls', args, {
+        env: process.env,
+        stdio: ['pipe', null, 'pipe'],
```

Comment:
I think you can omit the `stdio` option.

Reply:
Done